ui/src/components/settings/PageStorage.svelte

```diff
@@ -130,6 +130,12 @@
                                 <input type="text" id={uniqueId} required bind:value={s3.endpoint} />
                             </Field>
                         </div>
+                        <div class="col-lg-12">
+                            <Field class="form-field form-field-toggle" name="s3.forcePathStyle" let:uniqueId>
+                                <input type="checkbox" id={uniqueId} required bind:checked={s3.forcePathStyle} />
+                                <label for={uniqueId}>Force Path Style?</label>
+                            </Field>
+                        </div>
```

Comment:
This checkbox should be optional and not required.
Let's also move it below under all the other S3 fields and make it a regular checkbox (aka. remove the `form-field-toggle` class).
Let's also remove the question mark from the label and add a info icon with tooltip explaining what this option do.
Or in other words, place the below code, before `<!-- margin helper -->`
```html
// add the tooltip action in the imports section
import tooltip from "@/actions/tooltip";

....

<div class="col-lg-12">
    <Field class="form-field" name="s3.forcePathStyle" let:uniqueId>
        <input type="checkbox" id={uniqueId} bind:checked={s3.forcePathStyle} />
        <label for={uniqueId}>
            <span class="txt">Force path-style addressing</span>
            <i
                class="ri-information-line link-hint"
                use:tooltip={{
                    text: 'Enable to force the request to use path-style addressing, eg. "https://s3.amazonaws.com/BUCKET/KEY" instead of the default "https://BUCKET.s3.amazonaws.com/KEY".',
                    position: "top",
                }}
            />
        </label>
    </Field>
</div>
```